docs/rest-api/rest-zeppelin-server.md

```diff
@@ -33,6 +33,46 @@ If you work with Apache Zeppelin and find a need for an additional REST API, ple
 
 ## Zeppelin Server REST API list
 
+### Get Zeppelin version
+  <table class="table-configuration">
+    <col width="200">
+    <tr>
+      <td>Description</td>
+      <td>This ```GET``` method returns Zeppelin version</td>
+    </tr>
+    <tr>
+      <td>URL</td>
+      <td>```http://[zeppelin-server]:[zeppelin-port]/api/version```</td>
+    </tr>
+    <tr>
+      <td>Success code</td>
+      <td>200</td>
+    </tr>
+    <tr>
+      <td> Fail code</td>
```

Comment:
there are extra space in here, next line and "sample JSON response" (<-- and a newline there)
could you check?